ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -12,13 +12,16 @@ nginx_sites:
       hosts: formplayer
       port: "{{ formplayer_port }}"
       method: "hash $cookie_sessionid consistent"
+   mappings:
+    - name: "$digest_submission"
+      variable: "$http_authorization"
```

Comment:
If this header is included (after 401) then the key for the rate limiting will be "authorization_remote_ip".

If it isn't (before 401) then the key is "".